### PR TITLE
footer: Link to our to be launched Discord

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Link to our to be launched Discord

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/_includes/components/chat.njk
Comment (ZJvandeWeg) [inline src/_includes/components/chat.njk:1]: Not sure if a chat bubble is clear enough, or it should be the discrd logo. However, I didn't know how to do that. :)

File: src/_includes/layouts/base.njk [hunk 1]
```diff
@@ -294,12 +294,17 @@ eleventyComputed:
                               {% include "components/github.njk" %}
                             </a>
                         </li>      
-                          <li class="inline md:m-0 md:block">
+                        <li class="inline md:m-0 md:block">
+                            <a href="/https://discord.gg/BK3nhCQN" class="block px-3 py-3 md:p-0">
```
Comment (gdziuba) [inline src/_includes/layouts/base.njk:298]:
```suggestion
                            <a href="/https://discord.gg/2RrvW8dkrF" class="block px-3 py-3 md:p-0">
```

Comment (gdziuba) [reply src/_includes/layouts/base.njk:298]: This is a permanent link.  The previous one I gave expired after 7 days.

File: src/_includes/layouts/base.njk [hunk 2]
```diff
@@ -294,12 +294,17 @@ eleventyComputed:
                               {% include "components/github.njk" %}
                             </a>
                         </li>      
-                          <li class="inline md:m-0 md:block">
+                        <li class="inline md:m-0 md:block">
+                            <a href="/https://discord.gg/2RrvW8dkrF" class="block px-3 py-3 md:p-0">
```
Comment (Yndira-FlowForge) [inline src/_includes/layouts/base.njk:298]:
```suggestion
                            <a href="https://discord.gg/2RrvW8dkrF" class="block px-3 py-3 md:p-0">
```

File: src/_includes/components/chat.njk
Comment (Yndira-FlowForge) [reply src/_includes/components/chat.njk:1]: I'll add the discord logo 😉

**Comment by gdziuba** [comment]
image
image

**Review by Yndira-FlowForge — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
@gdziuba Lets update the footer when you have an updated link. It's a one-liner you can do too 😃

**Comment by gdziuba** [comment]
> @gdziuba Lets update the footer when you have an updated link. It's a one-liner you can do too 😃

@ZJvandeWeg Do you want me to boost the server for a custom link?  Let's ee the growth and change later?

**Pull request merged**